dashboard/datacenter.py

```diff
@@ -125,10 +125,10 @@ async def get_node_info(cls, node_id, get_summary=False):
         node = DataSource.nodes.get(node_id, {})
 
         if get_summary:
-            node_stats.pop("coreWorkersStats", None)
-        else:
             node_physical_stats.pop("workers", None)
             node_stats.pop("workersStats", None)
+        else:
```

Comment:
so there was a bug? was this caught by the existing test?

Reply:
yeah, it was